### PR TITLE
Add Submariner 2023 review

### PR DESCRIPTION
**Pull request by dfarrell07**
Fixes: submariner-io/submariner#2501

### CODE REVIEW

**Comment by dfarrell07** [comment]
Marked as draft while the community reviews.

**Review by skitt — APPROVED** [review]

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* Maxim Babushkin (@MaxBab, Red Hat)
+* pkomarov (@pinikomarov, unknown)
+* Steve Mattar (@SteveMattar, ex-Red Hat)
+* Yossi Boaron (@yboaron, Red Hat)
+
+https://github.com/orgs/submariner-io/people
+
+## Adoption
+
+Submariner has adopters actively using it in production for a verity of usecases.
```
Comment (tpantelis) [inline reviews/sandbox/2023/2023-submariner-annual.md:65]:
```suggestion
Submariner has adopters actively using it in production for a variety of use cases.
```

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+We haven't discussed identifying users, but see the Add Public Adopters goal.
+
+https://github.com/submariner-io/submariner/blob/devel/ADOPTERS.md
+
+## Performance Against Past Goals
+
+### Added Adopters
+
+One of Submariner's goals was to increase our number of adopters. We've accomplished this to a significant extent.
```
Comment (tpantelis) [inline reviews/sandbox/2023/2023-submariner-annual.md:74]:
```suggestion
One of our goals for Submariner was to increase the number of adopters. We've accomplished this to a significant extent.
```

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+We seem to have quite a few users doing a verity of interesting things with Submariner and subsets of Submariner.
+We'll continue trying to grow Submariner's user community.
+
+### Didn't Add Committers from not-Red Hat
```
Comment (tpantelis) [inline reviews/sandbox/2023/2023-submariner-annual.md:78]:
```suggestion
### Didn't Add Committers outside of Red Hat
```

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+One of Submariner's goals was to add Committers or Owners from groups other than Red Hat.  We have not accomplished this goal.
```
Comment (tpantelis) [inline reviews/sandbox/2023/2023-submariner-annual.md:80]:
```suggestion
Another goal for Submariner was to add Committers or Owners from groups other than Red Hat.  We have not accomplished this goal.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: reviews/sandbox/2023/2023-submariner-annual.md (hunk 3 above)
Comment (dfarrell07) [reply reviews/sandbox/2023/2023-submariner-annual.md:78]: How about "from Groups other than"? I think that's a bit more clear and avoids the double negative.

File: reviews/sandbox/2023/2023-submariner-annual.md (hunk 4 above)
Comment (dfarrell07) [reply reviews/sandbox/2023/2023-submariner-annual.md:80]: I guess you're also trying to shift the ownership of the goal from SubM to "for SubM", which makes sense. I'll send something parallel to the first change.

File: reviews/sandbox/2023/2023-submariner-annual.md (hunk 3 above)
Comment (tpantelis) [reply reviews/sandbox/2023/2023-submariner-annual.md]: sure

**Review by nyechiel — APPROVED** [review]

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+### Added Adopters
+
+One of our goals for Submariner was to increase the number of adopters. We've accomplished this to a significant extent.
+We seem to have quite a few users doing a variety of interesting things with Submariner and subsets of Submariner.
+We'll continue trying to grow Submariner's user community.
+
+### Didn't Add Committers from Groups other than Red Hat
+
+Another goals for Submariner was to add Committers or Owners from groups other than Red Hat. We have not accomplished this goal.
```
Comment (tpantelis) [inline reviews/sandbox/2023/2023-submariner-annual.md:80]:
```suggestion
Another goal for Submariner was to add Committers or Owners from groups other than Red Hat. We have not accomplished this goal.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: reviews/sandbox/2023/2023-submariner-annual.md (hunk 5 above)
Comment (dfarrell07) [reply reviews/sandbox/2023/2023-submariner-annual.md:80]: Good catch, thanks.

**Review by tpantelis — APPROVED** [review]

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+Members:
+
+* Andrew Stoycos (@astoycos, Red Hat)
+* Brent Salisbury (@nerdalert, Red Hat)
+* Maayan Friedman (@maayanf24, Red Hat)
+* Nir Yechiel (@nyechiel, Red Hat)
+* Noam Manos (@manosnoam, Red Hat)
+* Masaki Kimura (@mkimuram, Hitachi)
+* Maxim Babushkin (@MaxBab, Red Hat)
+* pkomarov (@pinikomarov, unknown)
```
Comment (dfarrell07) [inline reviews/sandbox/2023/2023-submariner-annual.md:57]: @skitt said this member is a Red Hatter and I confirmed. Will send an edit to fix.

File: reviews/sandbox/2023/2023-submariner-annual.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+Members:
+
+* Andrew Stoycos (@astoycos, Red Hat)
+* Brent Salisbury (@nerdalert, Red Hat)
+* Maayan Friedman (@maayanf24, Red Hat)
+* Nir Yechiel (@nyechiel, Red Hat)
+* Noam Manos (@manosnoam, Red Hat)
+* Masaki Kimura (@mkimuram, Hitachi)
+* Maxim Babushkin (@MaxBab, Red Hat)
+* pkomarov (@pinikomarov, Red Hat)
```
Comment (skitt) [inline reviews/sandbox/2023/2023-submariner-annual.md:57]:
```suggestion
* Pini Komarov (@pinikomarov, Red Hat)
```

Comment (dfarrell07) [reply reviews/sandbox/2023/2023-submariner-annual.md:57]: Yeah I thought about that but I didn't see his name and handle associated publically (although I didn't look closely) and didn't want to cause a privacy issue.

Comment (dfarrell07) [reply reviews/sandbox/2023/2023-submariner-annual.md:57]: I guess his handle is his name, if you know how to split it. I'll just add it.

Comment (skitt) [reply reviews/sandbox/2023/2023-submariner-annual.md:57]: Yeah that’s why I made the suggestion — the handle reveals all ;-).

**Comment by dfarrell07** [comment]
> Could you add a section for most recent accomplishments for the past year? It was not clear of the achievements.
> 
> Thanks!

Thanks for the review @linsun! I can add that, but I don't see it in the [annual review docs](https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md#annual-review-contents). Should those docs be updated, or is there somewhere else requirements are coming from?

**Comment by linsun** [comment]
@dfarrell07 thank you for raising the concern, i think it is reasonable to have the annual review docs to be updated. I'll open an issue for discussion.

I found it helpful during annual review to understand what is new.  An example I just reviewed: https://github.com/cncf/toc/pull/1078/files

**Comment by linsun** [comment]
Actually - I saw this in the [annual review docs](https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md#annual-review-contents), I think that is pretty much what I was looking for.

> How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)

**Comment by amye** [comment]
Marking this as complete, annual reviews are no longer a requirement for sandbox. 
We'll be rolling out more efficient ways to show project health and metrics in 2024.

**Pull request merged**